nominees/a11y.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "a11yproject",
+    "description": "The A11Y Project is a community-driven effort to make digital accessibility easier.",
+    "website": "https://www.a11yproject.com/",
+    "license": [{
+        "spdx": "Apache-2.0",
+        "licenseURL": "https://github.com/a11yproject/a11yproject.com/blob/main/LICENSE-APLv2"
+    }],
+    "SDGs": [{
+            "SDGNumber": 8,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+        },
+        {
+            "SDGNumber": 9,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+
+        },
+        {
+            "SDGNumber": 12,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+
+        }
+    ],
+    "type": [
+        "software",
+        "data"
+    ],
+    "repositoryURL": "https://github.com/a11yproject/a11yproject.com",
+    "organizations": [{
+        "name": "The Accessibility",
+        "website": "https://www.a11yproject.com/contact/",
```

Comment:
For the website field, the domain is good enough, no need to point to the contact page. Kindly update to the main [domain](https://www.a11yproject.com). Thanks.